### PR TITLE
mm-update send-proactive-messages.md

### PR DESCRIPTION
**Pull request by mmiele**
Update note.

### CODE REVIEW

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md [hunk 1]
```diff
@@ -19,8 +24,9 @@ Proactive messages can be useful in a variety of scenarios. These are some examp
 ### Basic guidelines
```
Comment (clearab) [inline msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: I don't think this entire section applies to Teams. Proactive messaging in teams is typically a welcome message or notification in a 1:1 conversation, or a notification into a channel or group chat. Very rarely would a proactive message occur in the midst of an on-going conversation, or need to break into a dialog. Please remove this and replace with the opening from the existing proactive message article.

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md [hunk 2]
```diff
@@ -66,7 +76,6 @@ You can only install apps that are in your organizational app catalogue, or the
 See [Install apps for users](https://docs.microsoft.com/graph/teams-proactive-messaging) in the Graph documentation for complete details. There is also a [sample in .NET](https://github.com/microsoftgraph/contoso-airlines-teams-sample/blob/283523d45f5ce416111dfc34b8e49728b5012739/project/Models/GraphService.cs#L176).
 
 
-
 ## Example
```
Comment (clearab) [inline msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: I'm not sure I'm following the sample code here. There should be a clear distinction between:

- A proactive message that creates a new conversation thread in a channel
- A proactive message that is a reply to an existing conversation thread in a channel
- A proactive message that is to a 1:1 or group chat

The last two might be identical, and can be combined, but the text should be clear about where/when they apply.

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md [hunk 3]
```diff
@@ -7,7 +7,12 @@ ms.author: anclear
 ---
 # Send proactive messages
 
-In some situations, a bot may need to send to the user a message that is not related to the current conversation. These types of messages are called **proactive messages**.
+<!-- Update -->
+
+A bot may need to send a message to the user that is not an immediate response to an incoming message. These types of messages are called **proactive messages**.  
+
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (mmiele) [reply msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: Removed note.

[the remark this answers is not in the document]
Comment (mmiele) [reply msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: Closing conversation.

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md (hunk 2 above)
Comment (mmiele) [reply msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: Andrew, let me know when the correct code sample for Teams is available. I was just trying to "adapt" the sample from the Bot Framework. I appreciate if you can highlight the snippets I can import and the rationale behind. Thanks

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md (hunk 1 above)
Comment (mmiele) [reply msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: Made requested changes.

File: msteams-platform/bots/how-to/conversations/send-proactive-messages.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmiele) [reply msteams-platform/bots/how-to/conversations/send-proactive-messages.md]: Removed note.

**Pull request merged**